backend/grant/admin/views.py

```diff
@@ -192,16 +187,16 @@ def delete_user(user_id):
 
     db.session.delete(user)
     db.session.commit()
-    return None, 200
+    return {"message": "ok"}, 200
 
 
 @blueprint.route("/users", methods=["GET"])
-@endpoint.api(
-    parameter('page', type=int, required=False),
-    parameter('filters', type=list, required=False),
-    parameter('search', type=str, required=False),
-    parameter('sort', type=str, required=False)
-)
+@query({
+    "page": fields.Int(required=False, missing=None),
+    "filters": fields.List(fields.Str(), required=False, missing=None),
+    "search": fields.Str(required=False, missing=None),
+    "sort": fields.Str(required=False, missing=None)
+})
 @admin.admin_auth_required
 def get_users(page, filters, search, sort):
     filters_workaround = request.args.getlist('filters[]')
```

Comment:
^

Reply:
I could use @AMStrix 's input here. I'm fine with leaving the workaround in if its robust.

Reply:
The workaround should no longer be needed if the webargs parser can handle GET query style arrays `?filters[]=arbiter&filters[]=admin`

Reply:
Ooh, yup, it does!  https://webargs.readthedocs.io/en/latest/quickstart.html

```
    # Delimited list, e.g. "/?languages=python,javascript"
    "languages": fields.DelimitedList(fields.Str()),
```

Reply:
I may have to make some tiny changes to the client to support this new style of query params, but it'll be for the better IMO.

Reply:
We can also diddle with axios to get it to output arrays differently if that seems like a better approach: https://stackoverflow.com/questions/49944387/how-to-correctly-use-axios-params-with-arrays